### PR TITLE
fix for reworked ScrollableEmbed

### PR DESCRIPTION
**Pull request by Czechbol**
Implementing pumpkin-py/pumpkin-py#157

### CODE REVIEW

File: points/module.py [hunk 1]
```diff
@@ -85,9 +85,8 @@ async def points_leaderboard(self, ctx):
         await utils.Discord.delete_message(ctx.message)
 
         if len(embeds) > 1:
-            scrollable_embed = utils.ScrollableEmbed()
-            scrollable_embed.from_iter(ctx, embeds)
-            await scrollable_embed.scroll(ctx)
+            scrollable_embed = utils.ScrollableEmbed(ctx, embeds)
```
Comment (ArcasCZ) [inline points/module.py:88]: This if else is redundant due to your fix of empty / 1 item list

**Review by ArcasCZ — COMMENTED** [review]
Actually

File: points/module.py (hunk 1 above)
Comment (Czechbol) [reply points/module.py]: Yeah, true. Fixed

**Review by sinus-x — APPROVED** [review]
LGTM

**Pull request merged**